tests/test_experiment.py

```diff
@@ -185,7 +185,7 @@ class EchoHelloWorldEntity(entity.SmartSimEntity):
     """A simple smartsim entity that meets the `ExecutableProtocol` protocol"""
 
     def __init__(self):
-        super().__init__("test-entity", _mock.Mock())
+        super().__init__("test-entity")
```

Comment:
Looks like `_mock` is no longer used! Feel from to remove the import!

Reply:
removed